types/vuelidate/vuelidate-tests.ts

```diff
@@ -19,9 +19,11 @@ const contains = (param: string): CustomRule =>
 
 const mustBeCool3 = helpers.withParams(
     { type: 'mustBeCool3' },
-    (value) => !helpers.req(value) || value.indexOf('cool') >= 0
+    (value: any) => !helpers.req(value) || value.indexOf('cool') >= 0
```

Comment:
The `TypeScript@next` test appears to have `noImplicitAny` enabled, which broke on previously accepted code that I didn't change. I updated with an explicit `any` to allow the `TypeScript@next` test to pass.